### ISSUE TITLE
can't use staging server

### ISSUE THREAD

**Issue by MarcSN311** [JrCs/docker-letsencrypt-nginx-proxy-companion#584]
I'm having problems obtaining certificates, so I wanted to switch to the staging server to cicumvent rate limiting. 
The Problem is: the staging Server does not allow ACMEv1, simp_le does not work with ACMEv2. 

```
ACME server returned an error: urn:acme:error:unauthorized :: The client lacks sufficient authorization :: Account creation on ACMEv1 is disabled. Please upgrade your ACME client to a version that supports ACMEv2 / RFC 8555. See https://community.letsencrypt.org/t/end-of-life-plan-for-acmev1/88430 for details.
```

**Comment by Coolstuff14** [comment]
This is because Lets Encrypt has stopped support for staging using ACME v1 protocol on October 1st. 
https://community.letsencrypt.org/t/end-of-life-plan-for-acmev1/88430/2

This means the switch to a compliant simp_le needs to happen sooner then November 1st.

**Comment by buchdag** [comment]
Hi.

An ACMEv2 compatible version will be released and tagged in the next ~48 hours (probably way sooner if things works out ok).

This will be a pre-release and won't be merged to master yet as there is still some work to be done on `simp_le`'s side, but it should pass the whole test suite.

**Comment by MarcSN311** [comment]
Thanks for the great work and fast response. I fixed my problem and only got rate limeted once, so no pressure from my side ;)

**Comment by buchdag** [comment]
Tagged version `v1.12-rc.1` with ACME v2 compatibility is now available.

**Comment by buchdag** [comment]
Fixed by #590 

Tagged version `v1.12` with ACME v2 compatibility is now available.
